tracer/test/Datadog.Trace.BenchmarkDotNet.Tests/Datadog.Trace.BenchmarkDotNet.Tests.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Datadog.Trace\Datadog.Trace.csproj" />
+    <ProjectReference Include="..\Datadog.Trace.TestHelpers\Datadog.Trace.TestHelpers.csproj" />
+  </ItemGroup>
+  
+  <ItemGroup Condition="!$(TargetFramework.StartsWith('netcoreapp3')) AND !$(TargetFramework.StartsWith('netcoreapp2'))">
```

Comment:
This doesn't make sense IMO, as you're no longer testing the _only_ project you're supposed to be testing 😉 Rather than this, and the other `#if`, I strongly suggest changing the TFMs instead.

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3774/commits/6325464f33bcf5fa227d094997d59350b1841a49